temoa/extensions/discrete_capacity/components/discrete_capacity.py

```
@@ -0,0 +1,108 @@
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from pyomo.environ import Constraint, quicksum, value
+
+import temoa.components.geography as geography
```

Ruff (0.15.21)

[warning] Use `from temoa.components import geography` in lieu of alias

Replace with `from temoa.components import geography`

(PLR0402)